python/test/t_CorrelationAnalysis_std.py

```diff
@@ -23,31 +24,30 @@
 inputSample = randomVector.getSample(sampleSize)
 outputSample = analytical(inputSample)
 
-src = ot.CorrelationAnalysis.SRC(inputSample, outputSample)
-print("src=", src)
+# Create the CorrelationAnalysis object
+corr_analysis = ot.CorrelationAnalysis(inputSample, outputSample)
 
-# Src with normalize
-src_normalize = ot.CorrelationAnalysis.SRC(inputSample, outputSample, True)
-print("src with normalize=", src_normalize)
+squared_src = corr_analysis.computeSquaredSRC()
+ott.assert_almost_equal(squared_src, [0.901238,0.105409], 1e-5, 0.0)
```

Comment:
The same exact reference values as used in the C++ unit test should be used at this point.

Reply:
Done.